presto-main/src/main/java/com/facebook/presto/operator/IGroupedTopNBuilder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator;
+
+import com.facebook.presto.common.Page;
+import com.google.common.util.concurrent.ListenableFuture;
+
+import java.util.Iterator;
+
+public interface IGroupedTopNBuilder
```

Comment:
Squash the commit that does the rename into this one where it's introduced.

Reply:
Because I am changing the contents of the file and renaming of the file a the same time, I had to keep the 2 commits separate. If I squash the 2, github will not show the exact lines changed. Instead it will show as 1 delete and 1 add. Is that ok for you to review ? I was under the impression that it will make it harder to review.

Reply:
got it. makes sense.